recipes/tcl/all/conanfile.py

```diff
@@ -98,6 +120,26 @@ def yes_no(v): return "yes" if v else "no"
     def _patch_sources(self):
         apply_conandata_patches(self)
 
+        # Un-vendor sqlite
+        # The folder has the sqlite version number in it, so need to find it dynamically
+        for folder in os.listdir(os.path.join(self.source_folder, "pkgs")):
+            if self.options.with_sqlite:
+                # tdbcsqlite is the wrapper package for tcl, don't modify it
+                if "sqlite" in folder and "tdbc" not in folder:
+                    rm(self, "*.c", os.path.join(self.source_folder, "pkgs", folder, "compat"), recursive=True)
+                    rm(self, "*.h", os.path.join(self.source_folder, "pkgs", folder, "compat"), recursive=True)
+
+                    # Don't override SQLITE_* variables, causes link errors
+                    if Version(self.version) < "8.6.13":
+                        replace_in_file(self, os.path.join(self.source_folder, "pkgs", folder, "Makefile.in"), "-DSQLITE_API=MODULE_SCOPE", "")
+                    replace_in_file(self, os.path.join(self.source_folder, "pkgs", folder, "Makefile.in"), "-DSQLITE_EXTERN=", "")
```

Comment:
Can you give me some insight into these changes? I haven't been able to find the compilation logs for the failed case. Thanks :)

Reply:
As far as I can tell, it messes with SQLite's linkage. These are overridden in TCL's build, so these are just un-overriding it to the defaults.